deploy-manage/tools/snapshot-and-restore/manage-snapshot-repositories.md

```diff
@@ -4,4 +4,58 @@
\ No newline at end of file
+% Scope notes: Landing page for this containing links for Cloud and ECE
+
+Snapshot repositories allow you to back up and restore your Elasticsearch data efficiently. Whether you're using [Elastic Cloud](#elastic-cloud-hosted), [Elastic Cloud Enterprise (ECE)](#elastic-cloud-enterprise-ece), or managing your own [Elasticsearch deployment](#self-managed), configuring a snapshot repository ensures data security, long-term archiving, and seamless migration across environments.
+
+## Supported repository types
+
+### Elastic Cloud Hosted
+
+Elastic Cloud Hosted deployments automatically register the `found-snapshots` repository. Elastic Cloud Hosted deployments use this repository and the `cloud-snapshot-policy` to take periodic snapshots of your cluster. You can also use the `found-snapshots` repository for your own [SLM policies](/deploy-manage/tools/snapshot-and-restore/create-snapshots.md#automate-snapshots-slm) or to store searchable snapshots.
+
+The `found-snapshots` repository is specific to each deployment. However, you can restore snapshots from another deployment’s found-snapshots repository if the deployments are under the same account and in the same region. 
+
+Elastic Cloud Hosted deployments also support the following repository types:
+
+* [Azure](/deploy-manage/tools/snapshot-and-restore/ec-azure-snapshotting.md)
+* [Google Cloud Storage](/deploy-manage/tools/snapshot-and-restore/ec-gcs-snapshotting.md)
+* [AWS S3](/deploy-manage/tools/snapshot-and-restore/ec-aws-custom-repository.md)
+* [Source-only](/deploy-manage/tools/snapshot-and-restore/source-only-repository.md)
+
+For more details, refer to Managing snapshot repositories in [Elastic Cloud Hosted](/deploy-manage/tools/snapshot-and-restore/elastic-cloud-hosted.md).
+
+### Elastic Cloud Enterprise (ECE)
+
+Snapshot repositories are managed for your entire Elastic Cloud Enterprise installation and can be specified for an Elasticsearch cluster when you create or manage it.
+
+When a repository is specified, a snapshot is taken every 30 minutes by default. The interval can be adjusted on per deployment basis.
+
+Elastic Cloud Enterprise installations support the following Elasticsearch snapshot repository types:
+
+* [Azure](/deploy-manage/tools/snapshot-and-restore/azure-storage-repository.md)
+* [Google Cloud Storage](/deploy-manage/tools/snapshot-and-restore/google-cloud-storage-gcs-repository.md)
+* [AWS S3](/deploy-manage/tools/snapshot-and-restore/minio-on-premise-repository.md)
+
+:::{note}
+No repository types other than those listed are supported in the Elastic Cloud Enterprise platform, even if they are supported by Elasticsearch. 
+:::
+
+For more details, refer to Managing snapshot repositories in [Elastic Cloud Enterprise](/deploy-manage/tools/snapshot-and-restore/cloud-enterprise.md).
+
+### Self-managed
+
+If you manage your own Elasticsearch cluster, you can use the following built-in snapshot repository types:
```

Comment:
Maybe we need to integrate ECK in this section also and offer links to ECK sub-sections.
ECK is considered `self-managed` also. A self-managed orchestrator.